modules/ROOT/pages/intro-platform-hosting.adoc

```diff
@@ -0,0 +1,146 @@
+
+* EU Cloud
++
+In the EU Cloud, the control plane is physically hosted within the European Union.
+
+* MuleSoft Government Cloud
++
+MuleSoft Government Cloud is a FedRAMP-compliant instance of the control plane that’s managed by MuleSoft and is suitable for government agencies.
++
+See xref:gov-cloud::index.adoc[About Government Cloud] for more information.
+
+* Customer-Hosted Control Plane
++
+MuleSoft also enables you to host the control plane within your own data center. Anypoint Platform Private Cloud Edition is a version of Anypoint Platform that supports hosting the control plane within a user-managed data center. This enables you to control the environment where your assets and metadata are stored.
++
+Private Cloud Edition supports deploying to Mule runtime servers hosted in a standalone environment. You can host the control plane in a physical server, a virtual server, or a 3rd-party cloud environment such as Amazon Web Services. 
++
+See xref:private-cloud::/latest/index.adoc[About Anypoint Platform PCE] for more information.
++
+Some Anypoint Platform components are not supported in Private Cloud Edition. See the <<Support Matrix for Control Plane Hosting Options>> for details.
+
+=== Support Matrix for Control Plane Hosting Options
+
+Not all Anypoint Platform components are supported in all cloud environments. The following table shows which component is supported by each environment:
+
+[%header,cols="5*a"]
+|===
+| Anypoint Platform Component | US Cloud | EU Cloud | MuleSoft Government Cloud | Private Cloud Edition
+| API Designer | Y | Y | Y | Y
+| Anypoint Studio | Y | Y | Y | Y
+| Flow Designer | Y | Y | N | N
+| Anypoint Exchange | Y | Y | Y | Y
+| Anypoint API Community Manager* | Y | Y | N | N
+| Anypoint Partner Manager | Y | N | N | N
+| Access management | Y | Y | Y | Y
+| Analytics | Y | Y | N | N
+| Anypoint API Manager | Y | Y | Y | Y
+| Anypoint Runtime Manager | Y | Y | Y | Y
+| Anypoint Monitoring | Y | Y | N | N
+| Secrets manager | Y | Y | N | N
+| Anypoint Visualizer | Y | Y | N | N
+| Anypoint Security edge policies | Y | Y | N | N
+| Anypoint Security tokenization | Y | Y | N | N
+| CloudHub runtimes | Y | Y | Y | N
+| Runtime Fabric | Y | Y | N | N
+| Customer-hosted Runtimes | Y | Y | N | Y
+|===
+
+== Runtime Plane Hosting Options
+
+MuleSoft provides three different options for hosting the Anypoint Platform runtime plane:
+
+* CloudHub
++
+CloudHub is MuleSoft's cloud-based environment for hosting the Mule runtime server and related services. CloudHub enables you to deploy an API or a Mule application on a platform that’s managed by MuleSoft.
++
+CloudHub also provides high-availability, clustering and failover of your APIs and Mule applications and load balances traffic to them. See xref:runtime-manager::cloudhub.adoc[CloudHub] for more information.
++
+CloudHub can only be managed by the Cloud control plane in the same environment (US Cloud, EU Cloud, or GovCloud).
```

Comment:
Do you mean AWS GovCloud? If you mean MuleSoft, it must be "MuleSoft Government Cloud". And the others do not include capital for cloud:
```suggestion
CloudHub can be managed only by the version of the cloud control plane that exists in the same environment (US cloud, EU cloud, or MuleSoft Government Cloud).
```

Reply:
Fixed.